[2023-08-24T15:22:45.334+0100] {processor.py:157} INFO - Started process (PID=5369) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:22:45.335+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:22:45.336+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:22:45.335+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:22:45.340+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:22:45.339+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:22:45.340+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:22:45.340+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:22:45.340+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:22:45.351+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-08-24T15:23:54.910+0100] {processor.py:157} INFO - Started process (PID=5431) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:23:54.927+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:23:54.929+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:23:54.928+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:23:54.935+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:23:54.933+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:23:54.935+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:23:54.935+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:23:54.935+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:23:54.948+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-08-24T15:24:56.983+0100] {processor.py:157} INFO - Started process (PID=5501) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:24:56.986+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:24:56.987+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:24:56.987+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:24:56.990+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:24:56.989+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:24:56.990+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:24:56.990+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:24:56.990+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:24:57.000+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-08-24T15:25:52.776+0100] {processor.py:157} INFO - Started process (PID=5554) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:25:52.785+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:25:52.786+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:25:52.786+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:25:52.790+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:25:52.789+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:25:52.790+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:25:52.790+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:25:52.790+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:25:52.802+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-08-24T15:26:48.504+0100] {processor.py:157} INFO - Started process (PID=5605) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:26:48.523+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:26:48.524+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:26:48.524+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:26:48.528+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:26:48.527+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:26:48.528+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:26:48.528+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:26:48.528+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:26:48.538+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-08-24T15:27:41.600+0100] {processor.py:157} INFO - Started process (PID=5655) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:27:41.613+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:27:41.614+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:27:41.614+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:27:41.621+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:27:41.620+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:27:41.621+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:27:41.621+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:27:41.622+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:27:41.639+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2023-08-24T15:28:37.053+0100] {processor.py:157} INFO - Started process (PID=5717) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:28:37.082+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:28:37.083+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:28:37.082+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:28:37.087+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:28:37.087+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:28:37.088+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:28:37.088+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:28:37.088+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:28:37.101+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.052 seconds
[2023-08-24T15:29:30.146+0100] {processor.py:157} INFO - Started process (PID=5783) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:29:30.162+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:29:30.163+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:29:30.162+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:29:30.167+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:29:30.166+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:29:30.167+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:29:30.167+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:29:30.168+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:29:30.180+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-08-24T15:31:13.625+0100] {processor.py:157} INFO - Started process (PID=6018) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:31:13.637+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:31:13.638+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:31:13.637+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:31:13.641+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:31:13.640+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:31:13.641+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:31:13.641+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:31:13.641+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:31:13.651+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-08-24T15:32:19.777+0100] {processor.py:157} INFO - Started process (PID=6084) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:32:19.813+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:32:19.814+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:32:19.814+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:32:19.819+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:32:19.818+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:32:19.819+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:32:19.819+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:32:19.819+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:32:19.833+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2023-08-24T15:33:09.533+0100] {processor.py:157} INFO - Started process (PID=6145) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:33:09.573+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:33:09.574+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:33:09.573+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:33:09.578+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:33:09.577+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:33:09.578+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:33:09.578+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:33:09.578+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:33:09.591+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-08-24T15:34:07.158+0100] {processor.py:157} INFO - Started process (PID=6204) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:34:07.188+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:34:07.189+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:34:07.189+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:34:07.194+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:34:07.193+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:34:07.194+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:34:07.194+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:34:07.194+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:34:07.206+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.052 seconds
[2023-08-24T15:35:13.845+0100] {processor.py:157} INFO - Started process (PID=6276) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:35:13.883+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:35:13.884+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:35:13.883+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:35:13.888+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:35:13.887+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:35:13.888+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:35:13.888+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:35:13.889+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:35:13.901+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2023-08-24T15:36:13.611+0100] {processor.py:157} INFO - Started process (PID=6339) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:36:13.639+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:36:13.641+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:36:13.640+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:36:13.645+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:36:13.644+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:36:13.645+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:36:13.645+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:36:13.645+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:36:13.658+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2023-08-24T15:37:08.157+0100] {processor.py:157} INFO - Started process (PID=6392) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:37:08.193+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:37:08.194+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:37:08.193+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:37:08.198+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:37:08.198+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:37:08.199+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:37:08.199+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:37:08.199+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:37:08.215+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-08-24T15:38:15.219+0100] {processor.py:157} INFO - Started process (PID=6446) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:38:15.220+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:38:15.221+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:38:15.220+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:38:15.225+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:38:15.224+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:38:15.225+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:38:15.225+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:38:15.225+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:38:15.237+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-08-24T15:39:18.331+0100] {processor.py:157} INFO - Started process (PID=6510) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:39:18.333+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:39:18.335+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:39:18.334+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:39:18.339+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:39:18.338+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:39:18.339+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:39:18.339+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:39:18.339+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:39:18.351+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-08-24T15:40:26.570+0100] {processor.py:157} INFO - Started process (PID=6593) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:40:26.588+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:40:26.590+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:40:26.589+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:40:26.594+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:40:26.593+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:40:26.594+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:40:26.594+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:40:26.594+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:40:26.607+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-08-24T15:41:22.526+0100] {processor.py:157} INFO - Started process (PID=6647) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:41:22.566+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:41:22.567+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:41:22.567+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:41:22.571+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:41:22.570+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:41:22.572+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:41:22.571+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:41:22.572+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:41:22.584+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-08-24T15:42:13.043+0100] {processor.py:157} INFO - Started process (PID=6700) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:42:13.046+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:42:13.047+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:42:13.046+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:42:13.051+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:42:13.050+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:42:13.051+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:42:13.051+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:42:13.051+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:42:13.063+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-08-24T15:43:09.655+0100] {processor.py:157} INFO - Started process (PID=6809) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:43:09.656+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:43:09.657+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:43:09.657+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:43:09.661+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:43:09.660+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:43:09.662+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:43:09.661+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:43:09.662+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:43:09.674+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-08-24T15:44:04.734+0100] {processor.py:157} INFO - Started process (PID=6863) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:44:04.769+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:44:04.770+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:44:04.770+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:44:04.774+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:44:04.774+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:44:04.775+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:44:04.775+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:44:04.775+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:44:04.787+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2023-08-24T15:44:53.326+0100] {processor.py:157} INFO - Started process (PID=6910) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:44:53.329+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:44:53.330+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:44:53.330+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:44:53.335+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:44:53.334+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:44:53.335+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:44:53.335+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:44:53.336+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:44:53.348+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-08-24T15:45:45.985+0100] {processor.py:157} INFO - Started process (PID=6961) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:45:46.012+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:45:46.013+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:45:46.012+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:45:46.017+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:45:46.016+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:45:46.017+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:45:46.017+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:45:46.018+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:45:46.030+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2023-08-24T15:46:38.697+0100] {processor.py:157} INFO - Started process (PID=7011) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:46:38.700+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:46:38.701+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:46:38.701+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:46:38.706+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:46:38.705+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:46:38.706+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:46:38.706+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:46:38.706+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:46:38.719+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-08-24T15:47:32.789+0100] {processor.py:157} INFO - Started process (PID=7062) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:47:32.819+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:47:32.821+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:47:32.820+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:47:32.825+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:47:32.824+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:47:32.825+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:47:32.825+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:47:32.825+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:47:32.838+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2023-08-24T15:48:27.010+0100] {processor.py:157} INFO - Started process (PID=7130) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:48:27.024+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:48:27.025+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:48:27.025+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:48:27.029+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:48:27.028+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:48:27.030+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:48:27.030+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:48:27.030+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:48:27.042+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-24T15:49:18.402+0100] {processor.py:157} INFO - Started process (PID=7179) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:49:18.419+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:49:18.420+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:49:18.420+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:49:18.424+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:49:18.424+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:49:18.425+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:49:18.425+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:49:18.425+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:49:18.437+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2023-08-24T15:50:07.169+0100] {processor.py:157} INFO - Started process (PID=7225) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:50:07.171+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:50:07.172+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:50:07.172+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:50:07.176+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:50:07.175+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:50:07.176+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:50:07.176+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:50:07.177+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:50:07.188+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-08-24T15:50:58.631+0100] {processor.py:157} INFO - Started process (PID=7273) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:50:58.666+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:50:58.667+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:50:58.667+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:50:58.671+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:50:58.671+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:50:58.672+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:50:58.672+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:50:58.672+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:50:58.684+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
[2023-08-24T15:51:52.010+0100] {processor.py:157} INFO - Started process (PID=7330) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:51:52.013+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:51:52.014+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:51:52.013+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:51:52.018+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:51:52.017+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:51:52.018+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:51:52.018+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:51:52.018+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:51:52.030+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-08-24T15:52:40.026+0100] {processor.py:157} INFO - Started process (PID=7380) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:52:40.046+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:52:40.047+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:52:40.047+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:52:40.051+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:52:40.050+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:52:40.051+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:52:40.051+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:52:40.052+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:52:40.063+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-08-24T15:53:32.294+0100] {processor.py:157} INFO - Started process (PID=7494) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:53:32.328+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:53:32.329+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:53:32.329+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:53:32.334+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:53:32.333+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:53:32.334+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:53:32.334+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:53:32.335+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:53:32.347+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2023-08-24T15:54:38.094+0100] {processor.py:157} INFO - Started process (PID=7588) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:54:38.112+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:54:38.113+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:54:38.113+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:54:38.117+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:54:38.117+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:54:38.118+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:54:38.118+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:54:38.118+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:54:38.130+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2023-08-24T15:55:38.943+0100] {processor.py:157} INFO - Started process (PID=7639) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:55:38.963+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:55:38.964+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:55:38.964+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:55:38.968+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:55:38.967+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:55:38.968+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:55:38.968+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:55:38.969+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:55:38.981+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2023-08-24T15:56:32.310+0100] {processor.py:157} INFO - Started process (PID=7824) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:56:32.347+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:56:32.348+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:56:32.348+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:56:32.352+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:56:32.351+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:56:32.352+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:56:32.352+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:56:32.352+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:56:32.362+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2023-08-24T15:57:20.690+0100] {processor.py:157} INFO - Started process (PID=7878) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:57:20.722+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:57:20.724+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:57:20.723+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:57:20.728+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:57:20.727+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:57:20.728+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:57:20.728+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:57:20.728+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:57:20.741+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2023-08-24T15:58:12.559+0100] {processor.py:157} INFO - Started process (PID=7931) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:58:12.593+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:58:12.595+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:58:12.594+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:58:12.599+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:58:12.598+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:58:12.599+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:58:12.599+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:58:12.599+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:58:12.611+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2023-08-24T15:59:06.035+0100] {processor.py:157} INFO - Started process (PID=8004) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:59:06.053+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T15:59:06.054+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:59:06.054+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:59:06.058+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:59:06.057+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T15:59:06.058+0100] {logging_mixin.py:151} INFO - [2023-08-24T15:59:06.058+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T15:59:06.059+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T15:59:06.070+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-08-24T16:00:03.825+0100] {processor.py:157} INFO - Started process (PID=8221) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:00:03.847+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T16:00:03.848+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:00:03.848+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:00:03.852+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:00:03.851+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T16:00:03.852+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:00:03.852+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T16:00:03.853+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:00:03.865+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-08-24T16:00:58.406+0100] {processor.py:157} INFO - Started process (PID=8332) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:00:58.418+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T16:00:58.419+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:00:58.419+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:00:58.424+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:00:58.423+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T16:00:58.424+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:00:58.424+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T16:00:58.424+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:00:58.436+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-08-24T16:01:52.165+0100] {processor.py:157} INFO - Started process (PID=8413) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:01:52.178+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T16:01:52.180+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:01:52.179+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:01:52.183+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:01:52.183+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T16:01:52.184+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:01:52.184+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T16:01:52.184+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:01:52.195+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-08-24T16:02:45.393+0100] {processor.py:157} INFO - Started process (PID=8480) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:02:45.428+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T16:02:45.430+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:02:45.429+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:02:45.434+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:02:45.433+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T16:02:45.434+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:02:45.434+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T16:02:45.434+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:02:45.449+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2023-08-24T16:03:36.797+0100] {processor.py:157} INFO - Started process (PID=8550) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:03:36.822+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T16:03:36.823+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:03:36.823+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:03:36.827+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:03:36.827+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T16:03:36.828+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:03:36.828+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T16:03:36.828+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:03:36.841+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2023-08-24T16:04:31.370+0100] {processor.py:157} INFO - Started process (PID=8625) to work on /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:04:31.404+0100] {processor.py:829} INFO - Processing file /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-24T16:04:31.406+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:04:31.405+0100] {dagbag.py:539} INFO - Filling up the DagBag from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:04:31.411+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:04:31.410+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-24T16:04:31.411+0100] {logging_mixin.py:151} INFO - [2023-08-24T16:04:31.411+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-24T16:04:31.412+0100] {processor.py:841} WARNING - No viable dags retrieved from /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-24T16:04:31.426+0100] {processor.py:179} INFO - Processing /home/samuel/dev/airflow_learning/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
